go-controller/pkg/util/util.go

```diff
@@ -349,3 +350,31 @@ func UpdateNodeSwitchExcludeIPs(nbClient libovsdbclient.Client, nodeName string,
 
 	return nil
 }
+
+// discovery.EndpointSlice reports in the same slice all endpoints along with their status.
```

Comment:
nit: should start func name. Fix the same for `IsEndpointReady` - starts with lowercase.